src/apiServices/ClientService.ts

```diff
@@ -2,12 +2,36 @@ import { AuthService } from './AuthService'
 import { AnalysesService } from './AnalysesService'
 import { JwtTokensInterface, AnalyzeOptions } from '..'
 
+/**
+ * Main service exposed to outside
```

Comment:
We want a period after sentences here (And in other similar places without one). When the docs are generated, the newlines will be removed (perhaps unless there is an extra newline in between, showing that a paragraph break is desired). So for this, it would look like "

Main service exposed to outside Needs to be instatiated with username, password and toolName (optional) fields" but "

Main service exposed to outside. Needs to be instatiated with username, password and toolName (optional) fields." is preferred.

Also, "instatiated" should be "instantiated" :wink:

Reply:
thanks if you spot more grammatical errors let me know :D